### ISSUE TITLE
constant space items

### ISSUE THREAD

**Issue by es183923** [FelixKratz/SketchyBar#78]
It'd be nice to have items that do not change in size when its label changes.

**Comment by FelixKratz** [comment]
I wanted to do this for some time, it is implemented now 22fc9ecfbdbadfc6c666a46afa5aa1c2a972f92e.
Just set it via e.g.:
```bash
sketchybar -m set <name> width <width in points>
```
or equivalently
```bash
sketchybar -m batch --set <name> width=<width in points>
```
This keeps items at fixed lengths.

**Comment by FelixKratz** [comment]
Done f4b3cd7b44f70000d03020bfd05ad3813782f781
